src/windows/osd.c

```diff
@@ -402,64 +402,109 @@ ssize_t sendmsg(SOCKET sd, struct msghdr *msg, int flags)
 	goto fn_complete;
 }
 
+void sort_list_of_adapters(PIP_ADAPTER_ADDRESSES head)
+{
+	PIP_ADAPTER_ADDRESSES ptr = head, tmp = NULL, prev = NULL;
+	bool flag = false;
+	if (head) {
```

Comment:
either require head to be non-null, or convert this check to:
```
if (!head)
    return;
```
so the entire function isn't indented

Reply:
@shefty Done.